frontend/public/components/overview/index.tsx

```diff
@@ -977,7 +970,16 @@ export const OverviewPage = withStartGuide(
   }
 );
 
-/* eslint-disable no-unused-vars, no-undef */
+type FirehoseItem = {
```

Comment:
Should we define these firehose types in a more central location so that they can be used elsewhere in the future?

Reply:
Yeah, we should look at this (and provide better definitions than what I have here). We do have an issue tracking: CONSOLE-327